src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingService.java

```diff
@@ -261,6 +264,25 @@ public boolean accept(File dir, String name) {
      */
     @Override
     public void onCreate() {
+        PRNGFixes.apply();
+        SharedPreferences sp = PreferenceManager.getDefaultSharedPreferences(this);
+
+        // Make sure this is also done for existing installs. We can replace this check with
+        // {@link #isFirstStart()} after a while.
+        if (!sp.getBoolean("default_user_pw_set", false)) {
+            sp.edit().putBoolean("default_user_pw_set", true).commit();
+            char[] chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789".toCharArray();
```

Comment:
I'm adding lowercase, but not sure which special characters we should use. It should be save enough anyway.